### DISCUSSION TITLE
Scheduled Install software

### DISCUSSION THREAD

**Discussion by franga92** [PSAppDeployToolkit/PSAppDeployToolkit#766] [Ideas]
hi guys,

I thought, is it possible to add a function to schedule the installation? For example, when you start the deployment where it tells you "click continue " to continue with the installation, I would add another button that says "postpone by" and there you have the option to decide whether 1 hour, 2 hours...
what do you say?

**Comment by cyborgrd** [comment]
There is already a defer function it defers until SCCM or another Deployment tool starts it again. You can combine it with Trigger-AppEvalCycle

https://discourse.psappdeploytoolkit.com/t/function-to-trigger-and-application-evaluation-cycle-after-a-specified-time/552
As example:
With Trigger-AppEvalCycle -Time 24 it will start again after 24H

**Comment by mjr4077au** [comment]
I think adding anything like this is positioning the toolkit for failure. You'd need to extend timeouts in SCCM and/or Intune to allow this, and the suspended installation would block other installs from occurring.
